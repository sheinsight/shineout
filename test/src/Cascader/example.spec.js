import exampleTest from '../../example'

describe('Cascader[snapshot]', () => {
  exampleTest('Cascader')
})
