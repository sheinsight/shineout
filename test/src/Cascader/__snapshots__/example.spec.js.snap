// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cascader[snapshot] Snapshot test: Cascader[site/pages/components/Cascader/example-01-base.tsx] 1`] = `
<span>
  <div>
    <div>
      <span>
         
      </span>
      <a>
        <svg>
          <path />
        </svg>
      </a>
    </div>
  </div>
</span>
`;

exports[`Cascader[snapshot] Snapshot test: Cascader[site/pages/components/Cascader/example-03-hover.tsx] 1`] = `
<span>
  <div>
    <div>
      <span>
         
      </span>
      <a>
        <svg>
          <path />
        </svg>
      </a>
    </div>
  </div>
</span>
`;

exports[`Cascader[snapshot] Snapshot test: Cascader[site/pages/components/Cascader/example-04-disabled.tsx] 1`] = `
<span>
  <div>
    <div>
      <span>
         
      </span>
    </div>
  </div>
</span>
`;

exports[`Cascader[snapshot] Snapshot test: Cascader[site/pages/components/Cascader/example-05-lazyload.tsx] 1`] = `
<span>
  <div>
    <div>
      <span>
         
      </span>
    </div>
  </div>
</span>
`;

exports[`Cascader[snapshot] Snapshot test: Cascader[site/pages/components/Cascader/example-06-filter.tsx] 1`] = `
<div>
  <div>
    <div>
      Single
    </div>
    <div>
      <span>
        <div>
          <div>
            <span>
               
            </span>
            <a>
              <svg>
                <path />
              </svg>
            </a>
          </div>
        </div>
      </span>
    </div>
  </div>
  <div>
    <div>
      Multiple
    </div>
    <div>
      <span>
        <div>
          <div>
            <span>
               
            </span>
          </div>
        </div>
      </span>
    </div>
  </div>
</div>
`;

exports[`Cascader[snapshot] Snapshot test: Cascader[site/pages/components/Cascader/example-07-filter-widematch.tsx] 1`] = `
<div>
  <div>
    <div>
      Single
    </div>
    <div>
      <span>
        <div>
          <div>
            <span>
               
            </span>
            <a>
              <svg>
                <path />
              </svg>
            </a>
          </div>
        </div>
      </span>
    </div>
  </div>
</div>
`;

exports[`Cascader[snapshot] Snapshot test: Cascader[site/pages/components/Cascader/example-07-innertitle.tsx] 1`] = `
<span>
  <div>
    <div>
      <div>
        select city
      </div>
      <div>
        <div>
          <span>
             
          </span>
          <a />
        </div>
      </div>
      <div>
        <div>
          select city
        </div>
      </div>
    </div>
  </div>
</span>
`;

exports[`Cascader[snapshot] Snapshot test: Cascader[site/pages/components/Cascader/example-08-unmatch.tsx] 1`] = `
<span>
  <div>
    <div>
      <a>
        未匹配值aaa
      </a>
      <span>
         
      </span>
      <a>
        <svg>
          <path />
        </svg>
      </a>
      <a />
    </div>
  </div>
</span>
`;

exports[`Cascader[snapshot] Snapshot test: Cascader[site/pages/components/Cascader/example-09-renderlist.tsx] 1`] = `
<span>
  <div>
    <div>
      <span>
         
      </span>
      <a>
        <svg>
          <path />
        </svg>
      </a>
    </div>
  </div>
</span>
`;

exports[`Cascader[snapshot] Snapshot test: Cascader[site/pages/components/Cascader/example-10-final.tsx] 1`] = `
<div>
  <div>
    <div>
      Single
    </div>
    <div>
      <span>
        <div>
          <div>
            <span>
               
            </span>
            <a>
              <svg>
                <path />
              </svg>
            </a>
          </div>
        </div>
      </span>
    </div>
  </div>
  <div>
    <div>
      multiple
    </div>
    <div>
      <span>
        <div>
          <div>
            <span>
               
            </span>
          </div>
        </div>
      </span>
    </div>
  </div>
</div>
`;
